KBKeyboardObserver/KBKeyboardObserver.h

```diff
@@ -62,12 +78,20 @@
 - (id)initWithReferenceView:(UIView *)referenceView;
 
 /**
+ Returns keyboard type. Initially set to NO, so could result in false negative if the observer was initialized during keyboard changeFrame.
+ 
+ @return returns keyboard type
+ */
+@property (nonatomic, readonly, getter = isStandardKeyboard) BOOL standardKeyboard;
```

Comment:
we don't get UIKeyboardWillShowNotification and UIKeyboardDidShowNotification non-standard keyboard ( changed keyboard frame ). and UIKeybardWillHideNotification and UIKeyboardDidHideNotification  same.
so, I think to need standardKeyboard check.